src/eval-frame/components/reps/value-renderer.jsx

```diff
@@ -6,56 +6,61 @@ import ExpandableRep from "./rep-tree";
 import ErrorRenderer from "./error-handler";
 import HTMLHandler from "./html-handler";
 import TableRenderer from "./data-table-rep";
-import UserReps from "./user-reps-manager";
 
-import { isRowDf } from "./rep-utils/rep-type-chooser";
-import { serializeForValueSummary } from "./rep-utils/value-summary-serializer";
-import { getChildSummary } from "./rep-utils/get-child-summaries";
-
-import { IODIDE_EVALUATION_RESULTS } from "../../iodide-evaluation-results";
-
-function repChooser(value) {
-  if (UserReps.getUserRepIfAvailable(value)) {
-    return "USER_REGISTERED_RENDERER";
-  }
-  if (value && value.iodideRender instanceof Function) {
-    return "IODIDERENDER_METHOD_ON_OBJECT";
-  }
-  if (value instanceof Error) {
-    return "ERROR_REP";
-  }
-  if (isRowDf(value)) {
-    return "TABLE_REP";
-  }
-  return "DEFAULT_REP";
-}
+import { requestRepInfo } from "./request-rep-info";
```

Comment:
all the changes in this file are about using `requestRepInfo` rather than direct functions for serializing object descriptors. this is the key stepping stone that will eventually allow truly remote reps across iframes.